images/linux/scripts/installers/java-tools.sh

```diff
@@ -39,6 +39,9 @@ apt-get -y install adoptopenjdk-11-hotspot=\*
 
 # Set Default Java version.
 update-java-alternatives -s /usr/lib/jvm/adoptopenjdk-${DEFAULT_JDK_VERSION}-hotspot-amd64
+if isUbuntu16; then
+    sudo update-alternatives --set javac /usr/lib/jvm/adoptopenjdk-${DEFAULT_JDK_VERSION}-hotspot-amd64/bin/javac
```

Comment:
Looks like this change doesn't fix the root cause but just change default Java. Let's investigate `update-java-alternatives` doesn't switch Java properly

Reply:
Done.